package.json

```diff
@@ -18,7 +18,8 @@
     "test:unit": "playwright test --config=playwright.config.unit.ts",
     "postinstall": "husky install",
     "notify": "node --experimental-modules scripts/notify.mjs",
-    "update:blocksuite": "pnpm i --filter @affine/app --filter @affine/datacenter @blocksuite/blocks@nightly @blocksuite/store@nightly && pnpm i --filter @affine/app @blocksuite/editor@nightly"
+    "update:blocksuite": "pnpm i --filter @affine/app --filter @affine/datacenter @blocksuite/blocks@nightly @blocksuite/store@nightly && pnpm i --filter @affine/app @blocksuite/editor@nightly",
+    "delete:node_modules": "find . -name \"node_modules\" -exec rm -rf '{}' +"
```

Comment:
could add a shell script instead of in package.json?

Reply:
Make sense, how about changes in https://github.com/toeverything/AFFiNE/pull/900/commits/ff772e6024089a3f91b804ed673d64f0c403e6d3